specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -164,22 +163,22 @@ export const notificationRenderers: IR<
   'record-merge-succeeded'(notification) {
     const id = parseInt(notification.payload.new_record_id);
     const tableName = notification.payload.table;
-    const model = getModel(tableName);
-    const resource = React.useMemo(
-      () =>
-        typeof model === 'object' ? new model.Resource({ id }) : undefined,
-      [model, id]
-    );
+    const newRecordId = parseInt(notification.payload.new_record_id);
     return (
-      resource !== undefined && (
-        <>
-          {mergingText.mergingHasSucceeded()}
-          <div className="flex items-center gap-2">
-            <TableIcon label name={tableName} />
-            <FormattedResource asLink={true} resource={resource} />
-          </div>
-        </>
-      )
+      <>
+        {mergingText.mergingHasSucceeded()}
+        <div className="flex items-center gap-2">
+          <TableIcon label name={tableName} />
+          <p>{`${tableName} #${newRecordId}`}</p>
+          <Link.NewTab
```

Comment:
while this solves the problem here, a similar issue could happen in other places.
as a cleaner solution, I suggest undoing this change, and instead fixing `<FormattedResource />` to handle 404 gracefully, instead of displaying a 404 page

Reply:
Fixed in 408a413afc0698e170fd71cbc9e578c3a441a9af